### DISCUSSION TITLE
Removed from EPIC on github without my consent ?!

### DISCUSSION THREAD

**Discussion by Xcogames** [community/community#147331] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I got an email that said: 

Removed from the "Epic Games" organization | Removed from the "Epic Games" organization |  
-- | -- | --
Removed from the "Epic Games" organization
 

Who removed me and why ?

**Comment by karasowles** [comment]
Hi - the owners of the Epic Games organization making access updates aren't something anyone except them can speak to. However, according to their documented process you can re-apply for access by following these instructions: https://github.com/epicgames

If you have any other questions, please reach out to them directly.
